src/System.Linq/tests/ConsistencyTests.cs

```diff
@@ -22,16 +22,18 @@ public static void MatchSequencePattern()
                 typeof(Enumerable),
                 typeof(Queryable),
                  new[] {
-                     "ToLookup",
-                     "ToDictionary",
-                     "ToArray",
-                     "AsEnumerable",
-                     "ToList",
+                     nameof(Enumerable.ToLookup),
+                     nameof(Enumerable.ToDictionary),
+                     nameof(Enumerable.ToArray),
+                     nameof(Enumerable.AsEnumerable),
+                     nameof(Enumerable.ToList),
                      "Fold",
                      "LeftJoin",
-                     "Append",
-                     "Prepend",
-                     "ToHashSet"
+                     nameof(Enumerable.Append),
+                     nameof(Enumerable.Prepend),
```

Comment:
Out of curiosity, why were `Append` and `Prepend` not added to the `Queryable` set of operators?